apps/explorer/src/ui/Banner.tsx

```diff
@@ -41,13 +74,30 @@ export function Banner({
     variant,
     align,
     fullWidth,
+    spacing,
+    onDismiss,
 }: BannerProps) {
     return (
-        <div className={bannerStyles({ variant, align, fullWidth })}>
-            {icon && (
-                <div className="flex items-center justify-center">{icon}</div>
-            )}
-            {children}
+        <div className={bannerStyles({ variant, fullWidth, spacing })}>
+            <div
+                className={contentStyles({
+                    align,
+                })}
+            >
+                {icon && (
+                    <div className="flex items-center justify-center">
+                        {icon}
+                    </div>
+                )}
+                <div className="overflow-hidden break-words">{children}</div>
+            </div>
+            {onDismiss ? (
+                <div className={closeBtnStyles({ spacing })}>
```

Comment:
I feel like this should just be position absolute which would simplify the markup here quite a bit and let you avoid the changes of having a content div.

Reply:
Yeah, that's was the initial implementation but wanted to fix the case when close icon overlays the content. Can change it back though. Let me know @Jordan-Mysten